src/utils/createTokenUser.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * Creates a user object based on the data provided by the user.
+ *
+ * @param {Object} user - User object with data.
+ * @param {string} user.name - User's name.
+ * @param {string} user._id - User's identifier.
+ * @param {string} user.role - User's role.
+ * @param {string} user.email - User's email.
+ * @returns {Object} - User object with data for token creation.
+ */
+
```

Comment:
Looks great! We can further define what the shape of the return value is by using the `typedef` tag: [https://jsdoc.app/tags-typedef.html](https://jsdoc.app/tags-typedef.html)

```suggestion
/**
 * @typedef {Object} TokenUser - User object with data for token creation.
 * @property {string} name - User's name.
 * @property {string} userId - User's ObjectId from the Mongo database.
 * @property {string} role - User's role. Can be either 'admin' or 'user'.
 * @property {string} email - User's email.
 */

/**
 * Creates a user object based on the data provided by the user.
 *
 * @param {Object} user - User object with data.
 * @param {string} user.name - User's name.
 * @param {string} user._id - User's identifier.
 * @param {string} user.role - User's role.
 * @param {string} user.email - User's email.
 * @returns {TokenUser} - User object with data for token creation.
 */

```

Reply:
<img width="755" alt="jsDoc" src="https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/95896159/cb6f1116-8df1-4476-9f8d-45300a6effb4">